samples: openthread: add nRF21540 DK support (#3579)
nrfconnect/sdk-nrf: doc/nrf/ug_radio_fem.rst

They are also valid for cases where an application uses just one protocol, but benefits from features provided by MPSL.
To avoid conflicts, check the protocol documentation to see if it uses FEM support provided by MPSL.

> Work is underway to make the protocols shipped with |NCS| use FEM. At the moment, :ref:`ug_thread` already supports the :ref:`nrf21540 DK <nrf21540dk_nrf52840>`, except for multiprotocol.

Reviewer: ```suggestion
Work is underway to make the protocols shipped with |NCS| use FEM. 
At the moment, :ref:`ug_thread` supports the :ref:`nrf21540 DK <nrf21540dk_nrf52840>`, but there is no multiprotocol support yet.
```
